apps/angular/src/app/features/auth/registration/registration.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="registration">
+    <form [formGroup]="formGroup" (ngSubmit)="handleSubmit()" class="registration__form">
+        <mat-form-field>
+            <mat-label>First name</mat-label>
+            <input matInput formControlName="firstName">
+        </mat-form-field>
+        <mat-form-field>
+            <mat-label>Last name</mat-label>
+            <input matInput formControlName="lastName">
+        </mat-form-field>
+        <mat-form-field>
+            <mat-label>Email</mat-label>
+            <input matInput formControlName="email">
```

Comment:
Let's add `type="email"` [attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) to improve UX. Don't forget to add correct 'type' attribute for input elements where possible:
```suggestion
            <input matInput formControlName="email" type="email">
```

Reply:
I've noticed similar code in other components as well. Don't forget to apply the change there as well.